packages/sitecore-jss-nextjs/src/middleware/personalize-middleware.ts

```diff
@@ -224,8 +230,11 @@ export class PersonalizeMiddleware {
       return response;
     }
 
+    // Path is rewritten by previous middleware
+    const basePath = res?.cookies.get('sc_path') || pathname;
```

Comment:
there is no way to access it, except by using nextjs internal `x-middleware-rewrite` header, which they set, but I'm not sure that we need to rely on it, cause it's not documented

Reply:
Got it. I was also thinking that a cookie is not the best place for storing the path like it is for site (since the path changes every request, a header makes more sense). I see 2 options:
1. Re-use `x-middleware-rewrite`
2. Use a custom header (e.g. `x-sc-rewrite`) instead of cookie. If we do this, we should also set this in other places that we are doing rewrites (i.e. personalize middleware) for consistency.

Reply:
Right, make sense, more safe to use second option